barretenberg/cpp/src/barretenberg/ecc/fields/field_impl.hpp

```diff
@@ -691,14 +691,8 @@ template <class T> field<T> field<T>::random_element(numeric::RNG* engine) noexc
     constexpr field pow_2_256 = field(uint256_t(1) << 128).sqr();
     field lo;
     field hi;
-    *(uint256_t*)lo.data = engine->get_random_uint256();
-    *(uint256_t*)hi.data = engine->get_random_uint256();
-    lo.self_reduce_once();
-    lo.self_reduce_once();
-    lo.self_reduce_once();
-    hi.self_reduce_once();
-    hi.self_reduce_once();
-    hi.self_reduce_once();
+    lo = engine->get_random_uint256();
```

Comment:
I restored the version that Zac checked in a previous PR. Accidentally got in an unchecked PR in there